src/forms.js

```diff
@@ -1,74 +1,78 @@
-export default http => ({
-  list: args => getForms(http, args),
-  get: args => getForm(http, args),
-  update: args => updateForm(http, args),
-  delete: args => deleteForm(http, args),
-  create: args => createForm(http, args),
-  messages: {
-    get: args => getMessages(http, args),
-    update: args => updateMessages(http, args)
+export default http => new Form(http);
+
+class Form {
+  constructor(_http) {
+    this._http = _http;
+    this._messages = new FormMessages(_http);
   }
-})
 
-const getForms = (http, { page, pageSize, search, workspaceId } = {}) => {
-  return http.request({
-    method: 'get',
-    url: `/forms`,
-    params: {
-      page,
-      page_size: pageSize,
-      search,
-      workspace_id: workspaceId
-    }
-  })
-}
+  get messages() {
+    return this._messages;
+  }
 
-const getForm = (http, { uid }) => {
-  return http.request({
-    method: 'get',
-    url: `/forms/${uid}`
-  })
-}
+  create({ data } = {}) {
+    return this._http.request({
+      method: 'post',
+      url: `/forms`,
+      data
+    });
+  }
 
-const updateForm = (http, { uid, override, data } = {}) => {
-  let methodType = 'patch'
-  if (override) {
-    methodType = 'put'
+  delete({ uid } = {}) {
+    return this._http.request({
+      method: 'delete',
+      url: `/forms/${uid}`
+    });
   }
 
-  return http.request({
-    method: methodType,
-    url: `/forms/${uid}`,
-    data
-  })
-}
+  get({ uid } = {}) {
+    return this._http.request({
+      method: 'get',
+      url: `/forms/${uid}`
+    });
+  }
 
-const createForm = (http, { data } = {}) => {
-  return http.request({
-    method: 'post',
-    url: `/forms`,
-    data
-  })
-}
+  list({ page, pageSize, search, workspaceId } = {}) {
+    return this._http.request({
+      method: 'get',
+      url: `/forms`,
+      params: {
+        page,
+        page_size: pageSize,
+        search,
+        workspace_id: workspaceId
+      }
+    });
+  }
 
-const deleteForm = (http, { uid }) => {
-  return http.request({
-    method: 'delete',
-    url: `/forms/${uid}`
-  })
-}
+  update({ uid, override, data } = {}) {
+    const methodType = override ? 'put' : 'patch';
 
-const getMessages = (http, { uid }) => {
-  return http.request({
-    method: 'get',
-    url: `/forms/${uid}/messages`
-  })
+    return this._http.request({
+      method: methodType,
+      url: `/forms/${uid}`,
+      data
+    });
+  }
 }
 
-const updateMessages = (http, { uid, data }) => {
-  return http.request({
-    method: 'put',
-    url: `/forms/${uid}/messages`,
-    data
-  })
-}
+class FormMessages {
+  constructor(_http) {
+    this._http = _http;
+  }
+
+  get({ uid }  = {}) {
+    return this._http.request({
+      method: 'get',
+      url: `/forms/${uid}/messages`
+    });
+  }
+
+  update({ uid, data }  = {}) {
+    return this._http.request({
+      method: 'put',
+      url: `/forms/${uid}/messages`,
+      data
+    });
+  }
+}
```

Comment:
maybe it would be useful to add standard to the project to avoid code style differences (semicolons, empty lines at the end)

Reply:
I added a lint step in the build, so it will fail if it's not standard compliant, as well as an eslint file that extends standard so you can see while you're coding the mistakes you've made